src/typescript/07-functions/functions.md

```diff
@@ -157,26 +80,44 @@ npm run test
 base cost.  This solution uses [Array.prototype.reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce):
 
 @sourceref ./../../../exercises/typescript/07-functions/01-solution/src/dnaCost.ts
+@highlight 1-4
 
 You’ll notice that specifying a return type is not necessary.  This is
 because TypeScript can infer the return value from the arguments.
 
 The following is another
 valid solution:
 
+@sourceref ./../../../exercises/typescript/07-functions/01-solution/src/dnaCost2.ts
 @highlight 1-4
 
-```ts
-export function dnaCost(baseCost: number, ...sequences: string[]) {
-  let sum = baseCost;
-  sequences.forEach(sequence => sum += sequence.length);
-  return sum
-}
+</details>
+
+### `this` parameters
```

Comment:
`this` is not a parameter… maybe this should be “`this` value” or something like that?

Also, this section should be with the other content, or split into its own objective, not orphaned after an exercise.